router-bridge/src/planner.rs

```diff
@@ -1871,6 +1891,19 @@ ofType {
             insta::assert_snapshot!(schema);
         }
     }
+
+    #[tokio::test]
+    async fn heap_statistics() {
+        let planner =
+            Planner::<serde_json::Value>::new(SCHEMA.to_string(), QueryPlannerConfig::default())
+                .await
+                .unwrap();
+
+        let _subgraphs = planner.subgraphs().await.unwrap();
+        let statistics = planner.get_heap_statistics().await.unwrap();
+
+        println!("statistics: {statistics:?}");
```

Comment:
Does this need an assertion, or is the point just that the call above shouldn't panic on `.unwrap()`?

Reply:
there's no guarantee of deterministic heap usage, I can't assert on the returned value, so the only thing we really need to know is if the function returned a value